metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -136,3 +161,63 @@ export function getNodeAutoInstrumentations(
 
   return instrumentations;
 }
+
+export function getResourceDetectorsFromEnv(): Array<Detector | DetectorSync> {
+  const resourceDetectorsFromEnv =
+    process.env.OTEL_NODE_RESOURCE_DETECTORS?.split(',');
```

Comment:
This is not an official OTEL variable. Did you happen to check what other SIGs are doing for that?